src/core/loading/mod.rs

```diff
@@ -1,14 +1,13 @@
+use crate::AppState;
 use bevy::core::FrameCount;
-pub use bevy::prelude::*;
-
-use super::AppState;
+use bevy::prelude::*;
 
 pub(super) fn plugin(app: &mut App) {
     // Setup(s), update(s), teardown(s)
     app.add_systems(Update, frames_elapsed.run_if(in_state(AppState::Loading)));
 }
 
-// Based on: <https://github.com/bevyengine/bevy/blob/v0.14.0/examples/window/window_settings.rs#L56>
+/// Based on: <https://github.com/bevyengine/bevy/blob/v0.14.0/examples/window/window_settings.rs#L56>
```

Comment:
Doc comment for private function?

Reply:
Yes, for IDE support.

Reply:
Huh ok.

Reply:
I like my mouseovers to show pretty docs ;)